src/main/java/org/mule/tools/apikit/model/ScaffoldingConfiguration.java

```diff
@@ -73,13 +86,18 @@ public Builder withDomain(MuleDomain domain) {
       return this;
     }
 
-    public Builder withShowConsole(boolean showConsole) {
-      this.showConsole = showConsole;
+    public Builder withAccessories(ScaffoldingAccessories scaffoldingAccessories) {
```

Comment:
instead of set the entire ScaffoldingAccessories, you can add a method for each field, and initialise a ScaffoldingAccessories in this build